docs/user-guide.rst

```diff
@@ -90,6 +90,7 @@ The :class:`~response.HTTPResponse` object provides
     import urllib3
 
     # Making the request (The request function returns HTTPResponse object)
+    # Here we're using module's request method. See the note in above section for details.
```

Comment:
I did this because of shazow's [comment](https://github.com/urllib3/urllib3/issues/2223#issuecomment-842380950)

Reply:
If it's mentioned in another section above do we need it duplicated?

Reply:
Sure, I'll get rid of it. The section is right above it anyways.